tls/test_tls_handshake.py

```diff
@@ -121,53 +127,48 @@ def test_bad_sni(self):
         """
         self.start_all()
         hs12 = TlsHandshake()
-        hs12.sni = ["bad.server.name"]
-        hs12.host = "tempesta-tech.com"
-        # Tempesta must send a TLS alerts raising TLSProtocolError exception.
-        with self.assertRaises(tls.TLSProtocolError):
-            hs12.do_12()
-        self.assertEqual(self.oops.warn_count("bad.server.name"), 1, "Bad SNI isn't logged")
+        hs12.sni = "bad.server.name"
+        hs12.do_12()
+        self.oops_ignore = ['WARNING']
+        self.assertEqual(hs12.hs.state.state, 'TLSALERT_RECIEVED')
+        self.assertEqual(self.oops.warn_count("requested unknown server name '.server.name'"), 1,
```

Comment:
Does '.server.name' should be expected here? Seems like issue on Tempesta side.

Reply:
Fixed, i remove domain name - test passed